src/main.cpp

```diff
@@ -158,6 +158,14 @@ void copyFont(const QString& externalPathName, const QString& resourcePathName,
 
 int main(int argc, char* argv[])
 {
+    // print stdout to console if Mudlet is started in a console in Windows
```

Comment:
I think this new code should go ***after*** the `--help`/`--version` code and the updater Windows-only auto update/restart code at the point where this comment is {though the comment itself is slightly inaccurate as it should be after the auto-update restart code}:
```
     /*******************************************************************
      * If we get to HERE then we are going to run a GUI application... *
     *******************************************************************/
```

Reply:
Why's that?

Reply:
Because there is Windows-specific handler code for the `-h`/`--help` and `-v`/`--version` ***command-line*** options and if that is called into play it renders the actions of this PR irrelevant; similarly if the Windows auto-updating code is activated then that terminates the Mudlet application. OTOH if non of the above happens then what you are proposing will be used/is useful for the Windows case (so it make sense to delay doing it until it ***is*** useful) ...

Reply:
OK, but there's no need for the `--help / --version` workaround now that we found this solution, right?

Reply:
Yep it does work OK:

![selection_228](https://user-images.githubusercontent.com/110988/46455618-2ee71a80-c7ac-11e8-9f37-3099c6fb083b.png)